plugins/workspace-minimap/README.md

```diff
@@ -30,9 +49,35 @@ const minimap = new Minimap(workspace);
 minimap.init();
 ```
 
+```css
+.blockly-minimap {
+  position: absolute;
+  box-shadow: none;
+  width: 200px;
+  height: 150px;
+  top: 0px;
+  left: 50vw;
+}
+```
+
+### Configuration
+The minimap takes a workspace as input and it inherits all of its properites (RTL, Theme, etc.).
```

Comment:
I would change this to stay it inherits the RTL and Theme properties, instead of saying it inherits all properties. Maybe also clarify that this is so they don't need to be configured manually.